src/Tabs.tsx

```diff
@@ -1 +1,29 @@
-// implement a component here
+import { FC } from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  tabs: Tab[],
+  selectedTab: Tab,
+  handleTabselected: (tab: Tab) => void,
+}
+
+export const Tabs: FC<Props> = ({ tabs, selectedTab, handleTabselected }) => {
+  return (
+    <ul className="nav nav-tabs mb-3">
+      {tabs.map(tab => (
+        <li key={tab.id} className="nav-item">
+          <button
+            className={classNames({
+              'nav-link': true,
+              active: tab.id === selectedTab.id,
+            })}
```

Comment:
No need to specify static class in object
```suggestion
            className={classNames('nav-link', {
              active: tab.id === selectedTab.id,
            })}
```